packages/webvh/src/WebvhModuleConfig.ts

```diff
@@ -0,0 +1,22 @@
+import { injectable } from 'tsyringe'
+
+export interface WebvhModuleConfigOptions {
+  /**
+   * Base URL for the WebVH service
+   */
+  baseUrl?: string
+}
+
+@injectable()
+export class WebvhModuleConfig {
+  private options: WebvhModuleConfigOptions
+
+  public constructor(options?: WebvhModuleConfigOptions) {
+    this.options = options ?? {}
+  }
+
+  /** See {@link WebvhModuleConfigOptions.baseUrl} */
+  public get baseUrl() {
+    return this.options.baseUrl ?? 'https://webvh.io'
```

Comment:
What does this default domain do? Is the baseUrl for when you create DIDs? I think having no default value and allowing it to be passed when creating one makes sense.